scripts/deploy/nlp/deploy_inframework_triton.py

```diff
@@ -98,6 +98,20 @@ def get_args(argv):
         type=int,
         help="Pipeline parallelism size",
     )
+    parser.add_argument(
+        "-nlfps",
+        "--num_layers_in_first_pipeline_stage",
```

Comment:
Do we expect that other models will need different flags? I'm thinking if we should keep the version with dedicated flag for every param, or rather have a general-purpose flag for passing any extra params. The current variant is easier for user (they don't need to remember exact param name), but wouldn't scale well if the list of flags grow too much.

Reply:
So, this MegatronDeployable class here https://github.com/NVIDIA-NeMo/Export-Deploy/blob/onur/fix-inframework-load-issue/nemo_deploy/nlp/megatronllm_deployable.py#L166 now accepts a kwarg. It's flexible enough but I also added the params you might need to the script. 

I am also not sure if these flags will be needed by another model or not but for some of them, looks like we need it to load it and serve it.

If there is a use-case where the script doesn't include the params needed, the user can create a script and pass them to the megatrondeployable.